docs/source/blocks/index.md

```diff
@@ -1,41 +1,20 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  "description": "Volto features the Pastanaga Editor Engine, allowing you to visually compose a page using blocks."
+  "property=og:description": "Volto features the Pastanaga Editor Engine, allowing you to visually compose a page using blocks."
+  "property=og:title": "Blocks"
+  "keywords": "Volto, Plone, frontend, React, blocks"
 ---
 
 # Blocks
 
-- [Introduction](introduction.md)
-
-- [Anatomy](anatomy.md)
-
-- [Settings](settings.md)
-
-- [Edit Components](editcomponent.md)
-
-- [Extensions](extensions.md)
-
-- [SSR](ssr.md)
-
 ```{toctree}
-:maxdepth: 3
-:hidden: true
-
-./introduction.md
-./anatomy.md
-./settings.md
-./editcomponent.md
-./extensions.md
-./ssr.md
-
+:maxdepth: 1
+
+introduction
+anatomy
+settings
+editcomponent
+extensions
+ssr
 ```
-
-- [Introduction](./introduction.md)
-- [Anatomy](./anatomy.md)
-- [Settings](./settings.md)
-- [Edit components](./editcomponent.md)
-- [Extensions](./extensions.md)
-- [SSR](./ssr.md)
```

Comment:
I like this clean up. 
Would you change the title of page /blocks/introduction from just "Blocks" to something like "Blocks Introcuction".

Reply:
Done in [3ec8de6](https://github.com/plone/volto/pull/3225/commits/3ec8de68b875dffe4353c11fbac09733d3bfa1ba).